pkg/scheduler/algorithm/predicates/metadata_test.go

```diff
@@ -25,7 +25,7 @@ import (
 	"k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	schedulernodeinfo "k8s.io/kubernetes/pkg/scheduler/nodeinfo"
-	schedulertesting "k8s.io/kubernetes/pkg/scheduler/testing"
+	st "k8s.io/kubernetes/pkg/scheduler/testing"
```

Comment:
nit: revert this line. It makes it easier to identify where an identifier is coming from.

Reply:
I'm more lean towards `st` as the places where `schedulertesting`/`st` is used is already indented with several tabs, saving 14 letters each line can make the code more compact and readable.

Reply:
I'm not convinced that compact==more-readable. But since I don't think this is [standardized](https://github.com/golang/go/wiki/CodeReviewComments#imports), I'll leave the decision to you.